src/utils/configuration.go

```diff
@@ -47,6 +50,11 @@ type telemetryConfiguration struct {
 	otelServiceName string
 }
 
+type additionalHeaders struct {
+	headers    map[string]string
+	headersStr string
```

Comment:
is this required? it seems like not being used? If not, they we don't need additionalHeaders struct

Reply:
sorry, I was meant to say `headerStr` seems not being used of this struct

Reply:
I see, its being used to log value to console once we have parsed overrideHeaders

<img width="783" alt="image" src="https://github.com/Azure/aad-auth-proxy/assets/45012871/b30d70a0-f345-41ef-86a3-8984d656762a">